R/cat_add_data.R

```diff
@@ -7,15 +7,113 @@
 #' sources.
 #'
 #' @param x an object of class [elic_cat].
-#' @param mechanism
+#' @param data_source either a [`data.frame`][base::data.frame] or
+#' [`tibble`][tibble::tibble], a string with the path to a _csv_ or _xlsx_ file,
+#' or anything accepted by the [read_sheet()][googlesheets4::read_sheet]
+#' function.
+#' @param mechanism integer indicating what mechanism the data belongs to.
```

Comment:
Line 14: it should be a character string. We can also discuss if we want to allow an integer index, but for the moment it should be a character.